lib/java/ydoc-polyfill/src/main/java/org/enso/ydoc/polyfill/WebSocketShell.java

```diff
@@ -101,12 +110,20 @@ public static void main(String[] params) throws Exception {
       System.exit(1);
     }
 
-    try (var client = new WebSocketClient(inspect)) {
+    try (var client = new WebSocketShell(inspect)) {
       var code = files.get(0);
       var res = client.eval(code);
-      System.out.printf("Script %s evaluated to %s\nPress any key to stop...", code, res);
-      System.in.read();
-      System.out.println("Exiting");
+      System.out.printf("Script %s evaluated to %s\n", code, res);
+      var console = System.console();
+      for (int i = 0; i < 10; i++) {
+        var line = console.readLine("js> ");
```

Comment:
This console.readLine seems to work surprisingly well. It supports arrow movements and even provides history of previous entries. Isn't it time to replace `JLine` with it? Isn't the console actually backed by JLine (repackaged)?